package.json

```diff
@@ -9,6 +9,7 @@
   },
   "main": "index.js",
   "scripts": {
+    "install": "npm run build:dll",
```

Comment:
I think we only want to run `build:dll when` we upgrade to a new version of angular, is that right? That, or if we change any of the packages that are part of our `vendors` module in webpack.